### PR TITLE
[cleanup](fe) cleanup useless code.

### PR DESCRIPTION
**Pull request by jackwener**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 36.17 seconds
 load time: 481 seconds
 storage size: 17122137225 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230121140228_clickbench_pr_84875.html

File: fe/fe-core/src/main/java/org/apache/doris/common/ErrorCode.java [hunk 1]
```diff
@@ -21,8 +21,7 @@
 
 // Error code used to indicate what error happened.
 public enum ErrorCode {
-    // Try our best to compatible with MySQL's
-    ERR_HASHCHK(1000, new byte[]{'H', 'Y', '0', '0', '0'}, "hashchk"),
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/common/ErrorCode.java:25]: don't delete any code from this file~

**Review by morrySnow — COMMENTED** [review]
@morningman PTAL~

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by morningman — COMMENTED** [review]
LGTM

**Pull request merged**